examples/microcoap_server/README.md

```diff
@@ -58,11 +58,59 @@ Waiting for incoming UDP packet...
 The link-layer address in this case is "fe80::f8bf:2bff:fe01:9ea3", the only
 "scope: local" address set.
 
-Connect using libcoap CLI
-=========================
+Testing
+=======
+There are multiple external CoAP clients you can use to test the server on native.
+
+libcoap CLI
+-----------
 
 (replace "fe80::f8bf:2bff:fe01:9ea3" with your link-layer address)
 
 ```
 # coap-client coap://[fe80::f8bf:2bff:fe01:9ea3%tap0]/riot/board
```

Comment:
Just a hint here: for zsh users it is necessary to put the url in quotes `coap-client "coap://[fe80::f8bf:2bff:fe01:9ea3%tap0]/riot/board"`. If this also works for bashers, then I'd vote to change it in the README.